web/styles/typeahead.css

```diff
@@ -176,20 +177,20 @@
 
 .typeahead-image {
     display: inline-block;
-    height: 21px;
-    width: 21px;
+    height: 1.1052em; /* 21px at 19px font size at 14px em */
+    width: 1.1052em; /* 21px at 19px font size at 14px em */
     border-radius: 4px;
 
     /* For FontAwesome icons and zulip icons used in place of images for some users. */
-    font-size: 19px;
+    font-size: 1.3571em; /* 19px at 14px em */
```

Comment:
I'd move the `font-size` declaration up to the top of the style block, and then it'll be obvious to read just `/* 21px at 19px/1em */` on the height and width adjustments.